### DISCUSSION TITLE
username in webhook conditional

### DISCUSSION THREAD

**Discussion by gstorme** [netbox-community/netbox#9701] [Q&A] [answered]
Hi,

Can a conditional on a webhook be made based on username?
I'm trying to filter out changes made by a specific username, but can't get it working

```
{
    "and": [
        {
            "attr": "username",
            "negate": true,
            "value": "gstorme"
        }
    ]
}
```

**Comment by jeremystretch** [comment, marked as answer]
Currently, conditions can only be evaluated against webhook data. The user associated with the request isn't available, but that could be a feature request.

**Reply by gstorme** [reply]
👍 https://github.com/netbox-community/netbox/issues/9707
